python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -521,6 +567,124 @@ def runDataRef(self, patchRefsByFilter, subdir=""):
                                                 dataId=repoInfo.dataId)[0]
         self.verifyJob.write(verifyJobFilename)
 
+    def readParquetTables(self, dataRefList, dataset, repoInfo, dfDataset=None):
```

Comment:
This seems to be a near-duplicate of the version of this in `CoaddAnalysisTask`.  It seems a bit uncomfortable to have this implemented in multiple places.  Would it be possible to move this into a function in utils.py, or have `ColorAnalysisTask` also inherit from the other analysis tasks to avoid the duplication?